compliance/collection/main.go

```diff
@@ -154,21 +156,12 @@ func manageSendToSensor(ctx context.Context, cli sensor.ComplianceService_Commun
 	}
 }
 
-func manageNodeScanLoop(ctx context.Context, rescanInterval time.Duration, scanner nodeinventorizer.NodeInventorizer) <-chan *sensor.MsgFromCompliance {
+func manageNodeScanLoop(ctx context.Context, i intervals.NodeScanIntervals, scanner nodeinventorizer.NodeInventorizer) <-chan *sensor.MsgFromCompliance {
 	sensorC := make(chan *sensor.MsgFromCompliance)
 	nodeName := getNode()
 	go func() {
 		defer close(sensorC)
-		t := time.NewTicker(rescanInterval)
-
-		// first scan should happen on start
-		msg, err := scanNode(nodeName, scanner)
-		if err != nil {
-			log.Errorf("error running scanNode: %v", err)
-		} else {
-			sensorC <- msg
-		}
-
+		t := time.NewTicker(i.Initial())
```

Comment:
The NodeInterval already logs the initial interval returned. Is that enough for what you want to log, or is there anything else you need?

Reply:
I was thinking here about the confirmation that everything has started correctly and we are now waiting for the initial scan. The other log message presents a summary that is useful to confirm that the config has been loaded and parsed. The message I suggest to add here would be more useful for the debugging but I can imagine that seeing it in a debug bundle from a customer would allow us to better understand the situation - i.e., the ticker has started and now we just need to wait.

I am not treating adding this log message as a "must", so you can make the final decision.

Reply:
Adding the log line before we call `scanNode()` is informative.